File: Grader/grader.rb
#grader class
class Grader
    def initialize(name, assignment, grade)
        #name, grade, and assignment name as instance variables 
        @grader_name = name
        @grader_grade = grade
        @grader_assignment = assignment
    end
    #method to calculate grade
    def getGrade
        if @grader_grade.to_f >=90
            puts @grader_name + "has a grade of: A, for "+@grader_assignment
        elsif @grader_grade.to_f >=80
            puts @grader_name + "has a grade of: B,  for "+@grader_assignment
        elsif @grader_grade.to_f >=70
            puts @grader_name + "has a grade of: C, for "+@grader_assignment
        elsif @grader_grade.to_f >=60
            puts @grader_name + "has a grade of: D, for "+@grader_assignment
        else
            puts @grader_name + "has a grade of: F, for "+@grader_assignment
    end
end

#assigns variables to input
puts "Enter student name?"
name = gets
puts "Enter assignment name?"
assignment = gets
puts "What is the grade for "+ assignment+"? (in numbers): "
grade = gets

#calls class and method
myGrade = Grader.new(name, assignment, grade)
myGrade.getGrade()
end 

File: Grader/grader.py
import sys

#grader class
class Grader: 
    def __init__(self):
        #name, assignment name, and grade as constructor
        gname = name;
        gasmtName = asmtName;
        ggrade = grade;
        #method to calculate grade
    def getGrade(self, name, asmtName, grade):
        if float(grade) >= 90:
            print (name+ "'s grade for "+asmtName+ " is: A")
        elif float(grade) >=80:
            print (name+ "'s grade for "+asmtName+ " is: B")
        elif float(grade) >=70:
            print (name+ "'s grade for "+asmtName+ " is: C")
        elif float(grade) >=60:
            print (name+ "'s grade for "+asmtName+ " is: D")
        else:
            print (name+ "'s grade for "+asmtName+ " is: F")


#assigns input to variable 
name = raw_input("Enter student name? ")
asmtName = raw_input("Enter assignment name? ")

#Validates if user typed number
while True:
    try:
        grade = float(raw_input("What is the grade for "+asmtName+"? (in numbers): "))
        break
    except:
        print('This is not a number... try again')

#calls class and method
myGrade = Grader()
print(myGrade.getGrade(name, asmtName, grade))

